Content.Server/Anomaly/AnomalySystem.Behavior.cs

```diff
@@ -0,0 +1,87 @@
+using System.Linq;
+using Content.Shared.Anomaly.Components;
+using Content.Shared.Anomaly.Prototypes;
+using Content.Shared.Random;
+using Content.Shared.Random.Helpers;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.Manager;
+
+namespace Content.Server.Anomaly;
+public sealed partial class AnomalySystem
```

Comment:
Newline, also since there are no event subscriptions, I feel like moving the methods here to a `region` in the main class would make more sense
Also use proxy EntityManager methods

Reply:
dontthink proxy supports things for compreg could be wrong

Reply:
Done